package.json

```diff
@@ -30,9 +30,11 @@
     "object-assign": "^4.0.1"
   },
   "devDependencies": {
+    "babel-preset-es2015": "^6.14.0",
```

Comment:
why do we need these? `isPrototypeOf` has been in JavaScript since ECMAScript 3rd edition, so no babelification should be necessary here?